### ISSUE TITLE
AccessKit node bounds are never clipped

### ISSUE THREAD

**Issue by DataTriny** [slint-ui/slint#7382]
### Bug Description

When an element contained in a scroll view is only partially visible, its AccessKit `Node::bounds` should be set accordingly.

## To reproduce

- Run the crud example
- Scroll the list view a bit so that the first item is partially hidden
- Observe with an accessibility inspector that the first item's bounding box goes outside of the item's parent bounding box.

### Reproducible Code (if applicable)

```slint
Unmodified crud example exhibits the bug.
```

### Environment Details

- Slint Version: master branch
- Platform/OS: Windows 11
- Programming Language: Rust
- Backend/Renderer: winit

**Comment by tronical** [comment]
If I recall correctly, we discussed that children should retain their bounds but the clipping element should set `set_clips_children`.
